hummingbot/connector/exchange/probit/probit_utils.py

```diff
@@ -75,7 +75,7 @@ def convert_diff_message_to_order_book_row(message: OrderBookMessage) -> Tuple[L
 KEYS = {
     "probit_api_key":
         ConfigVar(key="probit_api_key",
-                  prompt="Enter your ProBit Client ID >>> ",
+                  prompt="Enter your ProBit API key >>> ",
```

Comment:
For this, we decided to use Client ID because that's what their UI suggest it was; a ClientID-Secret key pair

Reply:
Thank you for the clarification Daniel. I will revert the change.